Website/src/rendering/src/assets/css/components/Products/ProductDetails.css

```diff
@@ -1,18 +1,21 @@
 .product-details {
   &-hero {
-    @apply grid grid-cols-2 gap-3;
+    @apply grid grid-cols-1 gap-3;
+    @apply sm:grid-cols-2;
 
     .product {
       &-name {
-        @apply col-span-2 text-xl font-bold w-full;
+        @apply text-xl font-bold w-full;
+        @apply sm:col-span-2;
       }
 
       &-image {
-        @apply row-span-3;
+        @apply lg:row-span-3;
 
         .image-active {
           > div {
-            @apply relative pt-[100%];
+            @apply relative pt-[100%] -mx-[20px];
```

Comment:
Is `-mx-[20px]` there to counter `.shop-container` `mx-5`? Can we instead use `-mx-5` for that to keep using REMs?